Service/src/request_validator/schemas/util_schema.py

```diff
@@ -51,4 +52,21 @@ def validateBuyPower(buyPower, field, errors):
          'InvalidField' : field.name + ' must be an integer greater than 1 and less than 1000000'
       })
 
-   return errors
\ No newline at end of file
+   return errors
+
+
+def validatePosInt(integer, field, errors):
+   if type(integer) != int or integer <= 0:
+      errors.append({
+         'InvalidField' : field.name + ' must be a positive integer'
+      })
+
+   return errors
+
+
+def validateTransType(transType, field, errors):
+   print("TRANSTYPE IS " + transType)
```

Comment:
Didn't you have a special logging function?

Reply:
shit, this line should've been deleted. good catch